packages/io-services-cms-models/openapi/services-cms-schemas.yaml

```diff
@@ -321,3 +321,29 @@ ServiceTopic:
   required:
     - id
     - name
+InternalServiceTopic:
+  type: object
+  properties:
+    id:
+      type: number
+      description: The topic id
+      example: 3
+    name:
+      type: string
+      description: The topic name
+      example: 'Benessere sociale'
+    deleted:
+      type: boolean
+      description: if the topic is deleted
+      example: false
+  required:
+    - id
+    - name
+    - deleted
+InternalServiceTopicList:
+  type: object
+  properties:
+    topics:
+      type: array
+      items:
+        $ref: '#/InternalServiceTopic'
```

Comment:
**chore:** move these definitions directly to cms-webapp openapi.yaml

Because this is a "temporary" workaround to expose data to a specific internal stakeholder (my2cents: in this Use Case good design is to push data when it changes, so change the logic from pull to push), there is no need to put these "temporary schemas" on commons packages.

Reply:
[5820204](https://github.com/pagopa/io-services-cms/pull/1037/commits/5820204e612de21762957ce817c8accfa2aa2f0f)